app/kuma-dp/pkg/dataplane/metrics/server.go

```diff
@@ -305,6 +315,7 @@ func (s *Hijacker) getStats(ctx context.Context, initReq *http.Request, app Appl
 	s.passRequestHeaders(req.Header, initReq.Header)
 	req = req.WithContext(ctx)
 	var resp *http.Response
+	logger.Info("executing get stats request", "address", app.Address, "port", app.Port, "path", app.Path)
```

Comment:
this should be V(1):

```suggestion
	logger.V(1).Info("executing get stats request", "address", app.Address, "port", app.Port, "path", app.Path)
```

Reply:
changed